common/scripted_triggers/wc_class_scripted_triggers.txt

```diff
@@ -761,11 +761,6 @@ is_alternative_class_trigger = {
 # You're <class>, but too weak to do something
```

Comment:
"You're a 'class', but too weak to do something" would make far more sense.

Reply:
It's a comment, only .csv files contain localization.